spec/factories/form526_submissions.rb

```diff
@@ -2,11 +2,13 @@
 
 FactoryBot.define do
   factory :form526_submission do
-    user_uuid { SecureRandom.uuid }
+    transient do
+      user { build(:disabilities_compensation_user) }
+    end
+    user_uuid { user.uuid }
```

Comment:
Use the `user`'s uuid (which is more realistic) rather than create an entirely random uuid.